static-assets/components/cstudio-forms/forms-engine.js

```diff
@@ -1470,6 +1470,7 @@ const initializeCStudioForms = () => {
 								CStudioAuthoring.Utils.showConfirmDialog({
 									body: formatMessage(formEngineMessages.formNotReadyForSaving)
 								});
+								setButtonsEnabled(true);
```

Comment:
One of the issues was that the form buttons we're not being enabled after running into a the 'Not ready for saving' error.